lib/galaxy/jobs/__init__.py

```diff
@@ -902,10 +902,16 @@ def get_special( ):
     def _create_working_directory( self ):
         job = self.get_job()
         try:
+            # TEMP BLOCK --- START
+            pluggedMedia = None
+            for pM in job.user.pluggedMedia:
```

Comment:
As marked, this part of the code is temporary. There will be some UI's asking user which `plugged_media` to use, or allow user to set a default `plugged_media` from preferences. However, till then, we're assuming that the user has only one `plugged_media` and we're retrieving it this way. 

regarding naming; indeed `job.user.plugged_media` is a relation to the `plugged_media` table.